adam/curriculum/phase1_curriculum.py

```diff
@@ -343,6 +353,58 @@ def _make_person_has_object_curriculum() -> Phase1InstanceGroup:
     )
 
 
+def _make_part_whole_curriculum() -> Phase1InstanceGroup:
+    whole_object_to_parts = {
+        BABY: ["head", "hand", "arm"],
+        BIRD: ["head", "wing"],
+        TRUCK: ["tire"],
+        CAR: ["tire", "trailer"],
+        DAD: ["head", "hand", "arm"],
+        MOM: ["head", "hand", "arm"],
+        DOG: ["head", "leg"],
+        HOUSE: ["wall", "roof"],
+    }
+    all_instances = []
+    for whole_object, parts in whole_object_to_parts.items():
+        whole = object_variable("whole", whole_object)
+
+        # Get the description sequence for "[whole] has a [part]" Using a part directly causes issues.
```

Comment:
Can you add a git issue tracking this hack for the curriculum so we remember this isn't using a template in the normal way?